src/angular-google-analytics.js

```diff
@@ -275,6 +280,9 @@ angular.module('angular-google-analytics', [])
               $window.ga('require', 'ecommerce', 'ecommerce.js');
             } else {
               $window.ga('require', 'ec', 'ec.js');
+              if(curency != 'USD'){
```

Comment:
Should use !== and there should be a space on either side of the conditional, if (...) {.